relay-config/src/config.rs

```diff
@@ -545,14 +545,13 @@ struct Http {
     /// events will be buffered.
     ///
     /// Defaults to `600` (10 minutes).
-    auth_interval: Option<u64>,
-    /// The time until Relay considers authentication dropped after experiencing errors.
+    auth_interval: Option<f64>,
```

Comment:
Can we please leave this as `u64`? I do not think it makes sense configuring this to milliseconds in any realistic case.

I'm assuming this has been changed for tests, and there we can accept that we have to wait 1s for this.

Reply:
Yes all changes from u64 to f64 have been done for testing, it does make a very big difference in the tests speed (for the few tests that deal with Upstream retries & re-auth).
As you wish, if you prefer to have slower tests but cleaner configuration I can change it back.